data/processing.py

```diff
@@ -640,11 +640,18 @@ def branch_for(agency):
     "Library of Congress",
     "The Legislative Branch (Congress)",
     "Government Printing Office",
-    "Congressional Office of Compliance"
+    "Government Publishing Office",
```

Comment:
The .gov list doesn't yet say `Government Publishing Office`, but I think they will on their next update. Better to future-proof this.